include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitMixedSignednessAccelerated;
+    //VkDeviceSize          storageTexelBufferOffsetAlignmentBytes;
+    //VkBool32              storageTexelBufferOffsetSingleTexelAlignment;
+    //VkDeviceSize          uniformTexelBufferOffsetAlignmentBytes;
+    //VkBool32              uniformTexelBufferOffsetSingleTexelAlignment;
+    //VkDeviceSize          maxBufferSize;
+
+    /* Vulkan Extensions */
+
+    /* ShaderCorePropertiesAMD *//* provided by VK_AMD_shader_core_properties */
+    //uint32_t           shaderEngineCount;
+    //uint32_t           shaderArraysPerEngineCount;
+    //uint32_t           computeUnitsPerShaderArray;
+    //uint32_t           simdPerComputeUnit;
+    //uint32_t           wavefrontsPerSimd;
+    //uint32_t           wavefrontSize;
+    //uint32_t           sgprsPerSimd;
+    //uint32_t           minSgprAllocation;
+    //uint32_t           maxSgprAllocation;
+    //uint32_t           sgprAllocationGranularity;
+    //uint32_t           vgprsPerSimd;
+    //uint32_t           minVgprAllocation;
+    //uint32_t           maxVgprAllocation;
+    //uint32_t           vgprAllocationGranularity;
+
+    /* ShaderCoreProperties2AMD *//* provided by VK_AMD_shader_core_properties2 */
+    //VkShaderCorePropertiesFlagsAMD    shaderCoreFeatures;
+    //uint32_t                          activeComputeUnitCount;
+
+    /* BlendOperationAdvancedPropertiesEXT *//* provided by VK_EXT_blend_operation_advanced */
+    //uint32_t           advancedBlendMaxColorAttachments;
+    //VkBool32           advancedBlendIndependentBlend;
+    //VkBool32           advancedBlendNonPremultipliedSrcColor;
+    //VkBool32           advancedBlendNonPremultipliedDstColor;
+    //VkBool32           advancedBlendCorrelatedOverlap;
+    //VkBool32           advancedBlendAllOperations;
```

Comment:
do not expose right now, no idea if we'll ever expose and implement those

but they'd all be `false` for OpenGL